src/Storage/Entity/ContentValuesTrait.php

```diff
@@ -397,6 +397,22 @@ public function setValues(array $values)
                 $this->values[$key] = $video;
             }
 
+            if ($this->fieldtype($key) == 'repeater' && is_array($this->values[$key]) && !$this->isRootType) {
```

Comment:
Ah, I just copied from L371. did a pass over that file, and they should all be strict now.